jobs/integr8ly/poc-testing-executor.yaml

```diff
@@ -42,8 +42,9 @@ This needs to be changed to 'false' for Integreatly Workshop since it uses valid
             default: master
             description: "Branch of a repository of Integreatly test suites"
         - string:
-            name: CLUSTER_URL
-            description: "URL of cluster for installation."
+            name: YOURCITY
```

Comment:
I know we discussed this on chat but looking at the PR I changed my mind (sorry). The `YOURCITY` is well-defined thing in PDS context but I don't think it is a good name in POC context. What we actually need is the cluster URL without the protocol. I think the best name would be CLUSTER_HOST (or CLUSTER_HOSTNAME)